src/content/docs/style-guide/punctuation.mdx

```diff
@@ -27,9 +27,13 @@ In writing, punctuation marks are used to imitate speech and play an important r
 -   Colons signal that what comes next is related directly to the previous word, phrase, or sentence.
 -   Never use a colon after a sentence fragment.
 
-### Examples
+General guidelines for using colons:
+
+##
 
--   Animal has 3 different styles for drumming: loud, louder, and deafening.
+-   In running text: Capitalize the first word after the colon if the word begins a complete sentence.
```

Comment:
```suggestion
-   In running text: Capitalize the first word after the colon if the word begins a complete sentence. Otherwise, do not capitalize the first word after a colon in running text.
```

Reply:
@bhecknr can you resolve this comment?